css/index.css

```diff
@@ -16,22 +16,70 @@ h1 {
     font-size: 50px;
     font-weight: bold;
     font-family: cedarville cursive;
-    
+
 }
 
 h2 {
     color: rgb(0, 255, 166);
     font-size: large;
-    font-weight: bold; 
+    font-weight: bold;
+}
+
+.job-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 5px;
+}
+
+
+#Experience {
+    padding: 20px;
+    margin: 20px;
+}
+
+#Experience h2 {
+    margin-bottom: 20px;
 }
 
+
 #Experience li {
     background-color: antiquewhite;
     box-shadow: 0 2px 4px rgba(144, 238, 144, 0.5);
+    padding: 10px;
+    margin-bottom: 10px;
+    border-radius: 5px;
+
+
 }
 
+.job-title {
+    font-weight: bold;
+}
+
+.job-description {
+    margin-left: 20px;
+    padding: 5px 0;
+}
```

Comment:
By applying these properties, you're making the job description a bit more to the right than the title instead of having it directly below. You can keep it that way if you want, it's just an observation about the alignment.